test/e2e/app-dir/next-image/next-image-https.test.ts

```diff
@@ -1,6 +1,7 @@
 import { nextTestSetup } from 'e2e-utils'
 
-describe('app dir - next-image (with https)', () => {
+// TODO: Test didn't (or maybe) never ran in CI but it should.
```

Comment:
@ztanner git-blame says you authored this test so I'm wondering if this regressed or never worked.